e2e/testdata/fn-render/out-of-place-dir/.expected/exec.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+set -eo pipefail
+
+# create a temporary directory
+TEMP_DIR=$(mktemp -d)
+
+kpt fn render -o $TEMP_DIR
+
+# copy the resources file back but retain Kptfile
+cp $TEMP_DIR/resources.yaml ./
+
+# remove temporary directory
+rm -r $TEMP_DIR
```

Comment:
This script will go away if we migrate to the change in my PR.

Reply:
Updated.